release.config.js

```diff
@@ -0,0 +1,18 @@
+module.exports = {
+  branches: [
+    '+([0-9])?(.{+([0-9]),x}).x',
+    'main'
+  ],
+  plugins: [
+    ['@semantic-release/commit-analyzer'],
+    '@semantic-release/release-notes-generator',
+    [
+      '@semantic-release/changelog',
+      {
+        changelogFile: 'CHANGELOG.md',
+      },
+    ],
+    '@semantic-release/git',
+    "@semantic-release/github",
```

Comment:
I'm not sure if you need both the `git` and `github` plugins, I think `github` may be capable of creating release tags.

Reply:
I will give it a shot without the git plugin, the github one was added later.

Reply:
:+1: right on!

Reply:
Works without the package so I will leave it out.